nautobot/ipam/urls.py

```diff
@@ -80,7 +80,7 @@
     path("rirs/<slug:slug>/edit/", views.RIREditView.as_view(), name="rir_edit"),
     path("rirs/<slug:slug>/delete/", views.RIRDeleteView.as_view(), name="rir_delete"),
     path(
-        "vrfs/<slug:slug>/changelog/",
+        "rirs/<slug:slug>/changelog/",
```

Comment:
Interesting! The only reason this was working, I think, is because for VRFs the pattern is instead `"vrfs/<uuid:pk>/changelog/"`, so this didn't actually conflict with the VRF changelog URLs.